main.js

```diff
@@ -545,3 +554,23 @@ function getCurrency (currencyString, currencySign) {
 	  }
 	});
 }
+function dirtyWords (wordToDefine) {
+	var url = "http://api.urbandictionary.com/v0/define?term="+wordToDefine;
+	request(url, function (err, response, body) {
+	 if(err){
+	    console.log('error getting definition: ', error);
+	  }
+	else {
+    	console.log('Urban Dictionary Definition:', body);
+    	var sample = JSON.parse(body);
+	var definition = sample.list[0]
```

Comment:
this still won't work

Reply:
just check if it exists before you use it

Reply:
it is tho line 567: definition != undefined